### ISSUE TITLE
Problem with fetch products from WooGraphql plugin by gatsby-source-wordpress

### ISSUE THREAD

**Issue by KryptonumDev** [gatsbyjs/gatsby#37734]
### Preliminary Checks

- [x] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I am writing an online store using gatsby-source-wordpress and WooGraphql plaugins.

Using the built-in WP Graphql IDE, all data comes in properly, however when I fetch data using gatsby-source-wordpress plugin attributes field comes up empty

I'm attaching a screenshot of the problem below.

![image_2023-03-10_15-12-05](https://user-images.githubusercontent.com/101868122/224338109-d8a8596a-8745-4356-b28e-a6f6db923eee.png)
![image_2023-03-09_15-31-27](https://user-images.githubusercontent.com/101868122/224338121-6a5817ab-bcb6-440d-9057-872934c77d2f.png)

### Reproduction Link

https://github.com/GleckoS/woocommerce

### Steps to Reproduce

1. Create gatsby project with gatsby-source-wordpress
2. Create wp instance with WPGraphQL | WP Gatsby | WooGraphQL plugins
3. Add product with attributes
4. Run local IDE

### Expected Result

Array of product attributes

### Actual Result

Empty array

### Environment

```shell
System:
    OS: Windows 10 10.0.19045
    CPU: (6) x64 Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz
  Binaries:
    Node: 18.12.1 - ~\AppData\Local\Temp\yarn--1678458004129-0.23139768955484974\node.CMD
    Yarn: 1.22.18 - ~\AppData\Local\Temp\yarn--1678458004129-0.23139768955484974\yarn.CMD
    npm: 8.19.2 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 3.10.4 - /c/Python310/python
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (110.0.1587.63)
  npmPackages:
    gatsby: ^5.7.0 => 5.7.0
    gatsby-plugin-advanced-sitemap: ^2.1.0 => 2.1.0
    gatsby-plugin-facebook-pixel: ^1.0.8 => 1.0.8
    gatsby-plugin-google-tagmanager: ^5.7.0 => 5.7.0
    gatsby-plugin-image: ^3.7.0 => 3.7.0
    gatsby-plugin-manifest: ^5.7.0 => 5.7.0
    gatsby-plugin-offline: ^6.7.0 => 6.7.0
    gatsby-plugin-react-helmet: ^6.7.0 => 6.7.0
    gatsby-plugin-react-svg: ^3.3.0 => 3.3.0
    gatsby-plugin-robots-txt: ^1.8.0 => 1.8.0
    gatsby-plugin-sharp: ^5.7.0 => 5.7.0
    gatsby-plugin-sitemap: ^6.7.0 => 6.7.0
    gatsby-plugin-styled-components: ^6.7.0 => 6.7.0
    gatsby-source-filesystem: ^5.7.0 => 5.7.0
    gatsby-source-wordpress: ^7.7.0 => 7.7.0
    gatsby-transformer-sharp: ^5.7.0 => 5.7.0
```

**Comment by KryptonumDev** [comment]
Also there is an error after updating product in wp, which disappear only after deleting gatsby-source-wordpress cache



info  gatsby-source-wordpress  update product FSO Polonez 1500 (#2797)
⠀
success  gatsby-source-wordpress  pull updates since last build - 3.730s
success Checking for changed pages - 0.001s
success source and transform nodes - 3.822s

 ERROR  UNKNOWN

Missing onError handler for invocation 'building-schema', error was 'Error: Cannot find ObjectTypeComposer with name WpProduct'. Stacktrace was 'Error: Cannot find ObjectTypeComposer with name WpProduct
    at SchemaComposer.getOTC (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\graphql-compose\src\SchemaComposer.ts:730:11)
    at forEach (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\schema\infer\index.js:30:37)
    at Array.forEach (<anonymous>)
    at addInferredTypes (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\schema\infer\index.js:27:13)
    at updateSchemaComposer (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\schema\schema.js:141:9)
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processTimers (node:internal/timers:504:9)
    at buildSchema (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\schema\schema.js:75:3)
    at build (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\schema\index.js:111:18)
    at buildSchema (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\services\build-schema.ts:19:3)'



sheva@DESKTOP-EG2P7IP MINGW64 ~/Desktop/projects/Auto-Welt (shop)
$ yarn start
yarn run v1.22.18
$ gatsby develop
success compile gatsby files - 3.022s
success load gatsby config - 0.069s
success load plugins - 1.772s
warn gatsby-plugin-react-helmet: Gatsby now has built-in support for modifying the document head. Learn more at https://gatsby.dev/gatsby-head
success onPreInit - 0.016s
success initialize cache - 0.070s
success copy gatsby files - 0.548s
success Compiling Gatsby Functions - 0.578s
success onPreBootstrap - 0.692s
success  gatsby-source-wordpress  ensuring plugin requirements are met - 0.006s
success  gatsby-source-wordpress  diff schemas - 2.029s
success  gatsby-source-wordpress  ingest WPGraphQL schema - 2.447s
success createSchemaCustomization - 2.650s
success  gatsby-source-wordpress  pull updates since last build - 1.193s
success  gatsby-source-wordpress  fetch root fields - 1.721s
success Checking for changed pages - 0.024s
success source and transform nodes - 2.451s
success building schema - 2.241s

 ERROR #11321  API.NODE.EXECUTION

"gatsby-node.js" threw an error while running the createPages lifecycle:

Interface field WpConnectionType.nodes expected but WpEnqueuedScriptConnectionType does not provide it.     

Interface field WpConnectionType.nodes expected but WpEnqueuedStylesheetConnectionType does not provide it. 

Interface field WpConnectionType.nodes expected but WpActionMonitorActionConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpPluginConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpThemeConnectionType does not provide it.

  2 |   const { createPage } = actions;
  3 |
> 4 |   const kolekcjeData = await graphql(`
    |                              ^
  5 |     {
  6 |       allWpKolekcje {
  7 |         edges {

File: gatsby-node.js:4:30



  Error: Interface field WpConnectionType.nodes expected but WpEnqueuedScriptConnectionType does not provide it.
  Interface field WpConnectionType.nodes expected but WpEnqueuedStylesheetConnectionType does not provide it.
  Interface field WpConnectionType.nodes expected but WpActionMonitorActionConnectionType does not provide it.
  Interface field WpConnectionType.nodes expected but WpPluginConnectionType does not provide it.
  Interface field WpConnectionType.nodes expected but WpThemeConnectionType does not provide it.

  - graphql-runner.ts:129 GraphQLRunner.validate
    [Auto-Welt]/[gatsby]/src/query/graphql-runner.ts:129:22

  - graphql-runner.ts:218 GraphQLRunner.query
    [Auto-Welt]/[gatsby]/src/query/graphql-runner.ts:218:49

  - create-graphql-runner.ts:57 apply
    [Auto-Welt]/[gatsby]/src/bootstrap/create-graphql-runner.ts:57:8

  - create-pages.ts:36 wrappedGraphQL
    [Auto-Welt]/[gatsby]/src/services/create-pages.ts:36:47

  - gatsby-node.js:4 Object.exports.createPages
    C:/Users/sheva/Desktop/projects/Auto-Welt/gatsby-node.js:4:30

  - api-runner-node.js:487 runAPI
    [Auto-Welt]/[gatsby]/src/utils/api-runner-node.js:487:37


failed createPages - 0.259s
success createPagesStatefully - 0.357s
info Total nodes: 1139, SitePage nodes: 39 (use --verbose for breakdown)
success Checking for changed pages - 0.002s
success write out redirect data - 0.006s
success Build manifest and related icons - 0.161s
success onPostBootstrap - 0.173s
info bootstrap finished - 18.979s
success onPreExtractQueries - 0.006s

 ERROR  UNKNOWN

Missing onError handler for invocation 'extracting-queries', error was 'Error: Interface field WpConnectionType.nodes expected but WpEnqueuedScriptConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpEnqueuedStylesheetConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpActionMonitorActionConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpPluginConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpThemeConnectionType does not provide it.'. Stacktrace was 'Error: Interface field WpConnectionType.nodes expected but WpEnqueuedScriptConnectionType does not provide it.    

Interface field WpConnectionType.nodes expected but WpEnqueuedStylesheetConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpActionMonitorActionConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpPluginConnectionType does not provide it.

Interface field WpConnectionType.nodes expected but WpThemeConnectionType does not provide it.
    at assertValidSchema (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\graphql\type\validate.js:59:11)
    at validate (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\graphql\validation\validate.js:63:35)
    at extractOperations (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\query\query-compiler.js:214:18)
    at processQueries (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\query\query-compiler.js:170:45)
    at compile (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\query\query-compiler.js:80:19)
    at updateStateAndRunQueries (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\query\query-watcher.ts:236:40)
    at extractQueries (C:\Users\sheva\Desktop\projects\Auto-Welt\node_modules\gatsby\src\services\extract-queries.ts:18:3)'

⠏ extract queries from components
